src/Command/Project/ProjectGetCommand.php

```diff
@@ -137,15 +137,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->debug('Initializing the project');
             $localProject->mapDirectory($projectRoot, $project);
 
+            if($git->getCurrentBranch($projectRoot) != $project->default_branch) {
+                $this->debug('current branch does not match the default_branch, create it.');
+                $git->checkOutNew($project->default_branch, null, null, $projectRoot);
+            }
```

Comment:
We should probably try to pass `--initial-branch` to the `git init` call above, maybe only if Git is version 2.28 or above.

https://git-scm.com/docs/git-init/2.28.0

Reply:
https://github.com/platformsh/platformsh-cli/pull/1077/files#diff-758a78d9bf121fd02f690e51541be9c098d2ad2ee91f1f991e885162b30a2b68R161 like this then?

Reply:
Exactly